examples/client/client.c

```diff
@@ -1401,7 +1401,7 @@ THREAD_RETURN WOLFSSL_THREAD client_test(void* args)
                    " nonblocking yet");
         } else {
             if (wolfSSL_Rehandshake(ssl) != SSL_SUCCESS) {
-                int  err = wolfSSL_get_error(ssl, 0);
+                err = wolfSSL_get_error(ssl, 0);
                 char buffer[WOLFSSL_MAX_ERROR_SZ];
```

Comment:
Good find on the "int err" here, but let's keep all variable declarations at top for compiler compatibility, so move the "char buffer[]" up a line. Thanks!